motoractuatorinterface.cpp

```diff
@@ -1,6 +1,20 @@
 #include "motoractuatorinterface.h"
+#include "constants.h"
```

Comment:
use line breaks to structure:
#include "motor..."

#include "constants.h"
#include "coord..system.h"